### PR TITLE
TC_00.000-04 | New Item > Create New item | New item from left Sidebar

### PR DESCRIPTION
**Pull request by Negrescu29**
### Added test case to add a new item from the sidebar. test run successfully locally 

**test case:** 
- https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/98

Screenshot 2024-11-18 at 21 48 20

### CODE REVIEW

**Review by wwwser11 — APPROVED** [review]

**Review by ols0test — APPROVED** [review]

**Review by ivanlashkevich — APPROVED** [review]
Ion, your original code works correctly. Next time try to apply a good practice of using 'and' for multiple assertions on the same element. LGTM. Approved

**Review by yanaoreshkevich — APPROVED** [review]
Approved

**Comment by Negrescu29** [comment]
Merge conflict resolved. all tests passed locally 
Screenshot 2024-11-19 at 11 01 31

File: cypress/e2e/newItemCreateNewItem.cy.js [hunk 1]
```diff
@@ -10,4 +14,17 @@ describe('US_00.000 | New Item > Create New item', () => {
...
+
+    it('TC_00.000-04 | New Item > Create New item | New item from left Sidebar', () => {
+        cy.get('span').contains('New Item').click()
+        cy.get('input#name').type(sidebarJobName)
+        cy.get('span.label').contains('Freestyle project').click();
+        cy.get('button#ok-button').click()
+        cy.get('button[name="Submit"]').click()
+        cy.get('.job-index-headline')
+            .should('contain.text', sidebarJobName)
+            .should('be.visible')
```
[the remark this answers is not in the document]
Comment (Negrescu29) [reply cypress/e2e/newItemCreateNewItem.cy.js:26]: Thanks @Rom-T great advice. will try to get an update for this test

**Pull request merged**